### DISCUSSION TITLE
评分弹窗有办法删除吗

### DISCUSSION THREAD

**Discussion by awsl2546** [the1812/Bilibili-Evolved#2519] [提问 / Q&A] [answered]
有没有在不关闭弹幕的情况下删除评分弹窗啊。

**Comment by Geeyun-JY3** [comment, marked as answer]
~~#2067 好像还没做？~~ V2 做了

V1 想要就参考：https://github.com/the1812/Bilibili-Evolved/discussions/2067#discussioncomment-1038442

**Reply by the1812** [reply]
刚看了下 v2 是支持的, v1 现在只维护不加新功能.

**Comment by awsl2546** [comment]
不好意思V2V1是什么意思，如果我现在需要这个功能要怎么实现啊。从控制台添加代码应该是一次性的吧，换一个页面是不是就要重新添加。

Grant Howard ***@***.***> 于2021年11月14日周日 下午4:47写道：

**Reply by Geeyun-JY3** [reply]
v2、v1 是版本，脚本设置页面和脚本管理器里都有些版本号的，看是 (v)1.x 还是 (v)2.x 就能判断了；v2 是还在预览的新版，是个大版本升级，用 v2 直接去设置项里开就行；还在 v1 就在控制台加代码，不是一次性的。

**Comment by awsl2546** [comment]
弄好了，感谢。

Geeyun-JY3 ***@***.***> 于2021年11月16日周二 上午10:07写道：
